src/code.sh

```
@@ -4,149 +4,260 @@
+    java -Xmx"${MAXHEAP}" -jar /usr/picard/picard.jar CollectVariantCallingMetrics \
+        --DBSNP="${DBSNP_VCF}" \
+        --INPUT="${VCF}" \
+        --OUTPUT="${OUTPUT_DIR}/${VCF_PREFIX}.variantcallingmetrics" \
+        --SEQUENCE_DICTIONARY="${SEQ_DICT}" \
+        --GVCF_INPUT true
 }
 
 main() {
+    ## Sanity checks
+    # Exit if no picard functions selected
+    if [[ "$run_CollectTargetedPcrMetrics" == "false" ]] && \
```

Shellcheck (0.10.0)

[warning] run_CollectTargetedPcrMetrics is referenced but not assigned.

(SC2154)